src/main/java/core/basesyntax/service/impl/TransactionProcessor.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.TransactionParser;
+import java.util.List;
+
+public class TransactionProcessor {
+    private final OperationStrategy operationStrategy;
+    private final TransactionParser transactionParser;
+
+    public TransactionProcessor(OperationStrategy operationStrategy,
+                                TransactionParser transactionParser) {
+        this.operationStrategy = operationStrategy;
+        this.transactionParser = transactionParser;
+    }
+
+    public void processTransactions(List<String> inputData) {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line at the start of the `processTransactions` method definition.